src/components/MovieList/MovieList.jsx

```diff
@@ -0,0 +1,111 @@
+import { Button, CardContainer } from 'components/MovieList/styles';
+import React, { useState } from 'react';
+
+import MovieCard from 'components/MovieCard/MovieCard';
+import PropTypes from 'prop-types';
+import blurImage from 'assets/image.jpg';
+import dayjs from 'dayjs';
+import { fetchDataFromApiList } from 'utils/function';
+
+// eslint-disable-next-line import/no-mutable-exports
+export let pageNumber = 1;
```

Comment:
I believe it is worth it to manage the `pageNumber` in a seperate state instead of using a global variable. Could we add a `pageNumber` state instead?  
  const [pageNumber, setPageNumber] = useState(1);